lib/query-builders/index.js

```diff
@@ -1,11 +1,12 @@
 const { get, isUndefined } = require('lodash');
 const Case = require('@ukhomeoffice/taskflow/lib/models/case');
+const { resolved, rejected, withdrawnByApplicant, discardedByApplicant } = require('../flow/status');
```

Comment:
There is a separate helper for getting closed/open tasks. Use that instead of re-enumerating here.

Reply:
https://github.com/UKHomeOffice/asl-workflow/blob/master/lib/flow/index.js#L77-L84